packages/terra-core-docs/src/terra-dev-site/doc/status/About.1.doc.mdx

```diff
@@ -9,8 +10,16 @@ import StatusPropsTable from 'terra-status/lib/Status?dev-site-props-table';
 
 <Badge />
 
+[![Deprecated](https://badgen.net/badge/status/Deprecated/grey)](https://github.com/cerner/terra-core/tree/main/packages/terra-status#readme)
+
 # Terra Status
 
+<Notice variant="deprecation">
+
+`terra-status` does not comply with accessibility guidelines and is no longer supported. If you are using `terra-status`, consider replacing it with `terra-icon` or `terra-image`.
```

Comment:
You would not replace status with a terra-icon or terra-image.  That would be the child of the component.  Consumers would have to create their own div to replace the terra-status component.